LICENSE.md

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright © `2017` `MediaCT`
+Copyright © `2017` `Youwe`
```

Comment:
I think year should be updated as well. For consistency could be removed, as you haven't added the year to all of the modules